### PR TITLE
Deliveroo Athena blog post

### PR DESCRIPTION
**Pull request by joshua-levin-deliveroo**

### CODE REVIEW

File: _posts/2022-08-05-athena-hackathon.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+layout: post
+title:  "See some ways Deliveroo champion women and non-binary people in tech"
+authors:
+- "Natalia Eremeeva"
+excerpt: >
+  Find out how Deliveroo's sponsorship of the Athena Hackathon supports women and non-binary talent in tech
+---
+
+## Hack on!
+In June 2022, Deliveroo was one of the main sponsors for the Athena Hackathon! The on-site event (with a virtual build-up), organised every year by ShowCode, aims to encourage, inspire and champion some of the amazing women and non-binary talent in the tech industry.
```
Comment (jamietanna) [inline _posts/2022-08-05-athena-hackathon.md:11]: Nit: something that changes between Markdown implementations is that a newline may be needed here:
```suggestion

In June 2022, Deliveroo was one of the main sponsors for the Athena Hackathon! The on-site event (with a virtual build-up), organised every year by ShowCode, aims to encourage, inspire and champion some of the amazing women and non-binary talent in the tech industry.
```

File: _posts/2022-08-05-athena-hackathon.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+We spent the hacking weekend speaking to prospective talent, encouraging the hackers, handing out Deliveroo Swag and providing technical mentorship with the support of our Data Science org.
+
+![Deliveroo representatives at the hackathon](/images/posts/athena-hackathon/f37c5006-09fc-4e2f-8eb2-8c1d9766f19b.JPG)
```
Comment (jamietanna) [inline _posts/2022-08-05-athena-hackathon.md:15]: is it worth lowercasing the extension here, as I'm not sure how well GitHub handles case-sensitive filenames?

File: _posts/2022-08-05-athena-hackathon.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+## Take a look at some of the key highlights from June's event
+
+- Over 600 women and non-binary techies (or wannabe techies!) registered
+- Over 50 projects submitted at the end of the hackathon
+- Over £10,000 in prizes distributed
+- Over 30 webinars and workshops
+
+Whilst it was fantastic to see the winning projects and watch the collective hard work come to fruition, overall participation in the event was the most important thing. The dedication and passion for technology and solution building was palpable and we proudly fuelled the good work (which ran into the small hours!) with our  “Dinner with Deliveroo” catering service. A huge well done to the team behind Data Saver Mode, a team of individuals who met and teamed up at the event, and also won the competition!
+
+![Hackathon winners](/images/posts/athena-hackathon/Winners.jpeg)
```
Comment (jamietanna) [inline _posts/2022-08-05-athena-hackathon.md:26]: as above re case sensitivity

File: _posts/2022-08-05-athena-hackathon.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Building the skills
+The Athena Hackathon was punctuated with workshops, webinars and tutorials and our very own engineers, engineering managers and data scientists who participated in the event and delivered presentations:
+
+Gina Wiley spoke about overcoming Imposter Syndrome
+<iframe width="560" height="315" src="https://www.youtube.com/embed/mCfT_Inyzmo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (jamietanna) [inline _posts/2022-08-05-athena-hackathon.md:32]: Let's enable privacy preserving embeds:
```suggestion
<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/mCfT_Inyzmo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment (jamietanna) [reply _posts/2022-08-05-athena-hackathon.md:32]: Do we also need the `allow`s?

File: _posts/2022-08-05-athena-hackathon.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+Mathilda Thompson gave a brief introduction to how the internet works
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Wre7qaMmiFo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Erika Morena Sierra spoke about her transition from IC to manager
+<iframe width="560" height="315" src="https://www.youtube.com/embed/nY8XqrRd_44" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Alina Nastasoiu explained how we tell stories with data at Deliveroo
+<iframe width="560" height="315" src="https://www.youtube.com/embed/sE0bFfwB-YA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (jamietanna) [inline _posts/2022-08-05-athena-hackathon.md:41]:
```suggestion

Gina Wiley spoke about overcoming Imposter Syndrome:

<iframe width="560" height="315" src="https://www.youtube.com/embed/mCfT_Inyzmo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

Mathilda Thompson gave a brief introduction to how the internet works:

<iframe width="560" height="315" src="https://www.youtube.com/embed/Wre7qaMmiFo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

Erika Morena Sierra spoke about her transition from IC to manager:

<iframe width="560" height="315" src="https://www.youtube.com/embed/nY8XqrRd_44" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

Alina Nastasoiu explained how we tell stories with data at Deliveroo:

<iframe width="560" height="315" src="https://www.youtube.com/embed/sE0bFfwB-YA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Maybe some newlines?

File: _posts/2022-08-05-athena-hackathon.md (hunk 4 above)
Comment (joshua-levin-deliveroo) [reply _posts/2022-08-05-athena-hackathon.md:32]: I copied these from youtube, i wasn't looking too much into the details.  happy to change it if it still works.

Comment (joshua-levin-deliveroo) [reply _posts/2022-08-05-athena-hackathon.md:32]: I just copied these form youtube, happy to change it if it is a drop-in replacement.

Comment (joshua-levin-deliveroo) [reply _posts/2022-08-05-athena-hackathon.md:32]: as for the allows - i presume that's to do with full screen stuff?

File: _posts/2022-08-05-athena-hackathon.md (hunk 2 above)
Comment (joshua-levin-deliveroo) [reply _posts/2022-08-05-athena-hackathon.md:15]: it seems fine both locally and live on the blog, i think we're okay.

**Pull request merged**